lib/simplifyExpression/fractionsSearch/cancelLikeTerms.js

```diff
@@ -287,6 +290,55 @@ function cancelTerms(numerator, denominator) {
     return new CancelOutStatus(numerator, denominator, true);
   }
 
+  // case 4: the numerator is a constant and denominator is a polynomial term
+  // e.g. 2 / 4x -> 1 / 2x
+
+  if (Node.Type.isConstant(numerator) && Node.PolynomialTerm.isPolynomialTerm(denominator)) {
+    const denominatorTerm = new Node.PolynomialTerm(denominator);
+
+    const coeff = denominatorTerm.getCoeffNode();
+    const variable = denominatorTerm.getSymbolNode();
+    const exponent = denominatorTerm.getExponentNode();
+
+    // simplify a constant fraction (e.g 2 / 4)
+    const frac = Node.Creator.operator('/', [numerator, coeff]);
+
+    let newCoeff;
+    const reduceStatus = divideByGCD(frac);
+
+    if (reduceStatus.hasChanged()) {
+      [numerator, newCoeff] = reduceStatus.newNode.args;
+    }
+    else {
+      return new CancelOutStatus(numerator, denominator, false);
+    }
+
+    denominator = Node.Creator.polynomialTerm(variable, exponent, newCoeff);
+
+    return new CancelOutStatus(numerator, denominator, true);
+  }
+
+  // case 5: both numerator and denominator are numbers
```

Comment:
We added this case to address this issue, right?: https://s3.amazonaws.com/socratic-ocr-uploads/prod/3a508ed6-a81b-4198-b647-ff59890d8e95/processed.jpeg

Maybe update the comment to include this example too? Otherwise it's unclear why we need this (because 2/4 by itself would get simplified somewhere else in the code base, wouldn't it?)